### PR TITLE
NXP-25709: Fix es hint field parsing

### PR DESCRIPTION
**Pull request by covolution**
Still waiting on a reliable test and push.

### CODE REVIEW

File: nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/query/NxqlQueryConverter.java [hunk 1]
```diff
@@ -97,20 +97,14 @@
  * visitor to build the ES request.
  */
 public final class NxqlQueryConverter {
+    protected static final int MORE_LIKE_THIS_MIN_TERM_FREQ = 1;
+    protected static final int MORE_LIKE_THIS_MIN_DOC_FREQ = 3;
+    protected static final int MORE_LIKE_THIS_MAX_QUERY_TERMS = 12;
```
Comment (bdelbosc) [inline nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/query/NxqlQueryConverter.java]: check again your indentation configuration, you should have emtpy lines

**Comment by bdelbosc** [comment]
:+1: when indentation is fixed

File: nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/query/NxqlQueryConverter.java (hunk 1 above)
Comment (covolution) [reply nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/query/NxqlQueryConverter.java]: Oh, i am not sure what happened there when i was stashing/unstashing. I will revert that.

**Comment by bdelbosc** [comment]
:+1:

**Comment by kevinleturc** [comment]
👍

**Comment by covolution** [comment]
Successfully T&P https://qa.nuxeo.org/jenkins/job/TestAndPush/job/ondemand-testandpush-gethin/161/

**Pull request merged**